static/js/compose.js

```diff
@@ -192,7 +192,7 @@ export function clear_compose_box() {
     if (compose_ui.is_full_size()) {
         compose_ui.make_compose_box_original_size();
     }
-    $("#compose-textarea").val("").trigger("focus");
+    $("#compose-textarea").trigger("focus").val("");
     compose_validate.check_overflow_text();
```

Comment:
We should also add a comment for this change in the code.

Reply:
oh this actually isn't needed anymore! thanks